doc/en/components/grids/_shared/excel-style-filtering.md

```diff
@@ -0,0 +1,1017 @@
+           github-src="{ComponentSample}/excel-style-filtering-sample-3" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-3"
+           github-src="{ComponentSample}/excel-style-filtering-sample-3" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
+
+Here is the full list of Excel style filtering components that you could use:
+- `Excel-style-header`
+- `Excel-style-sorting`
+- `Excel-style-moving`
+- `Excel-style-pinning`
+- `Excel-style-hiding`
+- `Excel-style-selecting`
+- `Excel-style-clear-filters`
+- `Excel-style-conditional-filter`
+- `Excel-style-search`
+
+
+## Unique Column Values Strategy
+
+The list items inside the Excel Style Filtering dialog represent the unique values for the respective column. These values can be provided manually and loaded on demand, which is demonstrated in the [{ComponentTitle} Remote Data Operations](remote-data-operations.md#unique-column-values-strategy) topic.
+
+## Formatted Values Filtering Strategy
+
+By default, the {ComponentTitle} component filters the data based on the original cell values, however in some cases you may want to filter the data based on the formatted values.
+
+<!-- ComponentStart: Grid, HierarchicalGrid -->
+
+In order to do that you can use the `FormattedValuesFilteringStrategy`.
+
+<!-- ComponentEnd: Grid, HierarchicalGrid -->
+
+<!-- ComponentStart: TreeGrid -->
+
+In order to do that you can use the `TreeGridFormattedValuesFilteringStrategy`.
+
+<!-- ComponentEnd: TreeGrid -->
+
+The following sample demonstrates how to format the numeric values of a column as strings and filter the {ComponentTitle} based on the string values:
+
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:620px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-formatted-filtering-strategy"
+           github-src="{ComponentSample}/formatted-filtering-strategy" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-formatted-filtering-strategy"
+           github-src="{ComponentSample}/formatted-filtering-strategy" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-formatted-filtering-strategy"
+           github-src="{ComponentSample}/formatted-filtering-strategy" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.